minix.sh

```diff
@@ -64,19 +64,59 @@ run_minix()
 }
 
 # Backupuje aktualny obraz
-# Tworzona jest kopia z nazwą zawierającą aktualną datę
-image_create_backup()
+# Tworzona jest kopia z nazwą użytkownika
+image_create_named_backup()
 {
+	if ! test -d ./backups/
+	then
+		mkdir -m 777 backups >/dev/null
```

Comment:
Po co 777?

Reply:
Inaczej usuwanie plików z tych folderów jako użytkownik wymaga wpisania komendy w konsoli jako sudo. Przynajmniej u mnie tak się dzieje. Jeśli uważasz, że tak powinno być, lub masz lepszy pomysł to można zmienić.

Reply:
No ale same pliki też są tworzone zdaje się z uid:gid root:root, więc i tak bez sudo nie da się ich ruszyć. Prędzej po zapisie backupu przywróciłbym mu usera który odpalił komendę. Samo polecenie chodzi jako root, więc i tak ma pełne prawa. Ustawianie 777 na czymkolwiek to zawsze zły pomysł.

Reply:
> Inaczej usuwanie plików z tych folderów jako użytkownik wymaga wpisania komendy w konsoli jako sudo. Przynajmniej u mnie tak się dzieje. 

U mnie się tak nie dzieje, po usunięciu tego `777` mam zarówno dostęp do folderu jak i plików wewnątrz. Właścicielem utworzonych plików i folderów jest użytkownik, a nie root.